src/components/BotMessageFeedback.tsx

```diff
@@ -49,7 +49,7 @@ function BotMessageFeedback({ message }: { message: CoreMessageType }) {
           }}
           disabled={!!message.myFeedback && message.myFeedback.rating !== FeedbackRating.GOOD}
         >
-          <Icon type={IconTypes.FEEDBACK_LIKE} width="24px" height="24px" />
+          <Icon type={'feedback-like'} size={24} />
```

Comment:
How about defining constant for the types?

Reply:
I've considered that option, but decided to use strings instead because it would make the code longer.

```tsx
const Types = {
  ICON_NAME: 'ICON_NAME,
}

<Icon type={Types.ICON_NAME} />
```

vs
```tsx
<Icon type={'icon-name'} />
```